internal/pkg/manifest/validate.go

```diff
@@ -2230,3 +2248,52 @@ func (i ImageLocationOrBuild) validate() error {
 	}
 	return nil
 }
+
+func validateConditionValuesPerRule(aliases []string, allowedSourceIPs []string) error {
+	if (len(aliases) >= maxConditionsPerRule) || (len(allowedSourceIPs) >= maxConditionsPerRule) ||
+		(len(aliases)+len(allowedSourceIPs) >= maxConditionsPerRule) {
+		return &errMaxConditionValuesPerRule{
+			aliases:          aliases,
+			allowedSourceIps: allowedSourceIPs,
+		}
+	}
+	return nil
+}
+
+type errMaxConditionValuesPerRule struct {
+	aliases          []string
+	allowedSourceIps []string
+}
+
+func (e *errMaxConditionValuesPerRule) Error() string {
+	return fmt.Sprintf("listener rule has more than five conditions %s %s", english.WordSeries(e.aliases, "and"),
+		english.WordSeries(e.allowedSourceIps, "and"))
+}
+
+func (e *errMaxConditionValuesPerRule) RecommendActions() string {
+	return fmt.Sprintf(`You can split the aliases in the manifest with "http.additional_rules" with the same path and container port:
```

Comment:
```suggestion
	return fmt.Sprintf(`You can split the "alias" and "allowed_source_ips" field into separate rules, so that each rule contains up to 5 values:
```